pkgs/build-support/build-bazel-package/default.nix

```diff
@@ -1,11 +1,12 @@
 { stdenv
-, bazel
 , cacert
 , lib
+, bazel
 }:
 
 args@{
   name
+, bazelVersion ? bazel
```

Comment:
I think `bazelVerson` gives the wrong hint (is it a string?). How about going with just `bazel`?

I've done the same change in https://github.com/NixOS/nixpkgs/pull/80953/commits/e5d8d31a3bb2c15e1e12a06db5e6911d0330221b

I like sticking to just `bazel` as that is the package name and is more intuitive for users already knowing the standard `.override { }` mechanism.

Reply:
Updated, I think this now matches your implementation?